### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Ali Zoubir Léonard Jouve

### DISCUSSION THREAD

**Discussion by Ali-Z0** [heig-vd-dai-course/heig-vd-dai-course#330] [Show and tell]
# Password manager with :

- Automatic or manual password generation
- Password information (Website)
- Encryption in a file 

## Private repository:
https://github.com/LeonardJouve/DAI-labo01.git
## Last commit 
91a91434b38287aac4f3ebaddffad3530c69d0c6
[commit](https://github.com/LeonardJouve/DAI-labo01/commit/91a91434b38287aac4f3ebaddffad3530c69d0c6)

**Comment by ludelafo** [comment]
Seems good to me but be wary of the available time. Focus on the essentials and good luck!

@GeraudSilvestri, do you agree?

**Comment by LeonardJouve** [comment]
Here is our latest [commit](https://github.com/LeonardJouve/DAI-labo01/commit/32fc3d5191627a12ff8341d274e101366ddb2eb4) as submition

**Reply by ludelafo** [reply]
Hello @LeonardJouve, @Ali-Z0,

Could you add @hadrylouis to your repository so he can grade your work as well?

Thank you,
Ludovic

**Reply by LeonardJouve** [reply]
It is done 👍
